app/src/main/java/com/zulip/android/ZulipApp.java

```diff
@@ -188,6 +212,7 @@ public void addToMutedTopics(JSONArray jsonArray) {
     public void setEmail(String email) {
         databaseHelper = new DatabaseHelper(this, email);
         this.you = Person.getOrUpdate(this, email, null, null);
+        setupEmoji();
```

Comment:
this seems unrelated to the `setEmail` call.  Can we move this to a more logical place?